config/crds/faros_v1alpha1_gittrack.yaml

```diff
@@ -6,6 +6,28 @@ metadata:
     controller-tools.k8s.io: "1.0"
   name: gittracks.faros.pusher.com
 spec:
+  additionalPrinterColumns:
```

Comment:
This is many columns, should we make some only be displayed in the wide format? (`-o wide`)